src/handlers/admin/handler.js

```diff
@@ -55,16 +55,17 @@ const commitMessageSync = async (kafkaTopic, consumer, message) => {
 
 const createRecordFundsInOut = async (payload, transactionTimestamp, enums) => {
   try {
+    /** @namespace Db.getKnex **/
+    const knex = Db.getKnex()
```

Comment:
Hey Georgi, I know this was done on a previous commit, but why are we using the knex object at this level of the code? should the queries not be in a facade layer?

Reply:
Hi Rajiv, your question is reasonable. Yet, if you examine the following couple of lines you'll see that it is only the transaction that is started here and the queries are written in the facade layer. This approach makes it possible to break the facade methods into smaller, more manageable and reusable pieces of code that can be later assembled to serve different purpose while executing in the same transaction (and it's used right away - check the sequence diagrams: seq-recfunds-5.1.0.*). I see your point and if you have a better solution, I'll be happy to discuss it.